src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdaterTest.java

```diff
@@ -123,50 +125,69 @@ public void createCaseBundlesPropertyWhenItDoesntExist() throws IOException {
         Mockito.when(mockRequest.getHeader("Authorization")).thenReturn("a");
         Mockito.when(mockRequest.getReader())
-                .thenReturn(
-                        new BufferedReader(
-                                new StringReader("{\"case_details\":{\"case_data\": {\"bundleConfiguration\":\"example.yaml\","
-                                        + "\"caseBundles\": "
-                                        + "[{\n"
-                                        + "\"value\": {\n"
-                                        + "\"id\": \"19c9f411-e9a0-43f4-af13-e3d7ee98a50b\",\n"
-                                        + "\"title\": \"New bundle\",\n"
-                                        + "\"description\": null,\n"
-                                        + "\"eligibleForStitching\": \"no\",\n"
-                                        + "\"eligibleForCloning\": \"no\",\n"
-                                        + "\"stitchedDocument\": null,\n"
-                                        + "\"documents\": [],\n"
-                                        + "\"folders\": [{\n"
-                                        + "\"value\": {\n"
-                                        + "\"name\": \"Folder 1 Original\",\n"
-                                        + "\"documents\": [],\n"
-                                        + "\"folders\": [{\n"
-                                        + "\"value\": {\n"
-                                        + "\"name\": \"Folder 1.a Original\",\n"
-                                        + "\"documents\": [],\n"
-                                        + "\"sortIndex\": 0\n"
-                                        + "}\n"
-                                        + "}],\n"
-                                        + "\"sortIndex\": 0\n"
-                                        + "}\n"
-                                        + "}],\n"
-                                        + "\"fileName\": \"original_bundle.pdf\",\n"
-                                        + "\"coverpageTemplate\": \"FL-FRM-APP-ENG-00002.docx\",\n"
-                                        + "\"hasTableOfContents\": \"Yes\",\n"
-                                        + "\"hasCoversheets\": \"Yes\",\n"
-                                        + "\"hasFolderCoversheets\": \"No\",\n"
-                                        + "\"stitchStatus\": null,\n"
-                                        + "\"paginationStyle\": \"topLeft\",\n"
-                                        + "\"pageNumberFormat\": \"numberOfPages\",\n"
-                                        + "\"stitchingFailureMessage\": null\n"
-                                        + "}\n"
-                                        + "}]}}}")
-                        ));
+            .thenReturn(
+                new BufferedReader(
+                    new StringReader("{\"case_details\":{\"case_data\": {\"bundleConfiguration\":\"example.yaml\","
```

Comment:
why don't we just use a test bundle file?

Reply:
This is case data.